bin/ds

```diff
@@ -527,6 +527,23 @@ function check_alias_param {
   fi
 }
 
+#=== FUNCTION ==================================================================
+# NAME: restart_php5_fpm
+# DESCRIPTION: Restarts php5-fpm on servers running php5.
+#===============================================================================
+function restart_php5_fpm {
+  PHP_FPM_STATUS="$(/etc/init.d/php5-fpm status)"
+  PHP_VERSION="$(php -v|grep --only-matching --perl-regexp "5\.\\d+\.\\d+")"
+
+  if [[ ${PHP_FPM_STATUS} == " * php5-fpm is running" ]] && $(echo ${PHP_VERSION} | grep --quiet "5.5")
+  then
+    echo "Restarting PHP-FPM now"
+    sudo /etc/init.d/php5-fpm restart
```

Comment:
We should only use `service` command in order to avoid exposing local environment variables:
`sudo service php5-fpm restart`